cpp/arcticdb/stream/stream_utils.hpp

```diff
@@ -404,22 +404,15 @@ storage::KeySegmentPair make_target_key(KeyType key_type,
     if (is_ref_key_class(key_type)) {
         return {RefKey{stream_id, key_type}, std::move(segment)};
     } else {
-        if (is_ref_key_class(variant_key_type(source_key))) {
```

Comment:
I currently cannot find an example where we copy a ref key to an atom key, so this feature is removed to avoid having to refactor out the content hash logic.